fabric-events-interaction-v0/build.gradle

```diff
@@ -1,3 +1,3 @@
 version = getSubprojectVersion(project)
 
-moduleDependencies(project, ['fabric-api-base'])
+moduleDependencies(project, ['fabric-api-base', 'fabric-networking-api-v1'])
```

Comment:
I believe this has to be declared on FMJ as well. see https://github.com/FabricMC/fabric/blob/1.21/fabric-screen-handler-api-v1/src/main/resources/fabric.mod.json

Reply:
Yes, good catch. Fixed.